wordlist.go

```diff
@@ -19,10 +19,10 @@ package dcrlibwallet
 import "strings"
 
 func PGPWordList() []string {
-	return strings.Split(alternatingWords, "\n")
+	return strings.Split(AlternatingWords, "\n")
 }
 
-const alternatingWords = `aardvark
+const AlternatingWords = `aardvark
```

Comment:
Why are we exporting this? I'd rather clients call `PGPWordList()` and concatenate the returned array to get a single string if necessary. dcrwallet does not export this string.

Reply:
`PGPWordList()` is not exported by gomobile.